src/it/jdk16-annotation/src/test/java/com/mycompany/jdk16annotation/AppTest.java

```diff
@@ -18,35 +18,20 @@
  */
 package com.mycompany.jdk16annotation;
 
-import junit.framework.Test;
-import junit.framework.TestCase;
-import junit.framework.TestSuite;
-
 /**
  * Unit test for simple App.
  */
 @ServiceProvider(service = App.class, path = "xxx", position = 1)
-public class AppTest extends TestCase {
+public class AppTest {
     /**
      * Create the test case
      *
      * @param testName name of the test case
      */
-    public AppTest(String testName) {
-        super(testName);
-    }
-
-    /**
-     * @return the suite of tests being tested
-     */
-    public static Test suite() {
-        return new TestSuite(AppTest.class);
-    }
+    public AppTest(String testName) {}
 
     /**
      * Rigourous Test :-)
      */
-    public void testApp() {
-        assertTrue(true);
```

Comment:
Keep this line after adding a `import static org.junit.jupiter.api.Assertions.assertTrue`.